deploy/template/etc/star_taskcluster-worker_net.crt

```diff
@@ -1,8 +1,37 @@
+Certificate for *.taskcluster-worker.net
+
+Used by workers to service HTTPS for livelog and interactive features.  These
+are saved to a .crt and .key file locally and used when deploying new AMIs
+(the locations of these are for the questions asked during the deploy process
+for ssl key location)
+
+webops can help renew this certificate
+
+-----BEGIN CERTIFICATE REQUEST-----
```

Comment:
The CSR doesn't need to be here.  The text above won't hurt, I think, but is also unnecessary.